Sources/Features/Metrics/Model/BacktraceMetricsSender.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+final class BacktraceMetricsSender {
+
+    private let api: BacktraceApi
+    private let metricsContainer: BacktraceMetricsContainer
+    private let settings: BacktraceMetricsSettings
+
+    static let defaultBaseUrl = "https://events.backtrace.io/api"
```

Comment:
I'd suggest storing all constant values in one place, so they can be easily updated/maintained

Reply:
Will do, thank you!